spec/list_spec.rb

```diff
@@ -25,8 +25,8 @@ def fake_xcodes(*names)
 
     describe '#list' do
       it 'lists all versions' do
-        fake_xcodes '1', '2.3', '3 some', '4 beta'
-        installer.list.should == "1\n2.3\n3 some\n4 beta"
+        fake_xcodes '1', '2.3', '3 some', '4 beta', '10 beta'
```

Comment:
should also test with a version that's `major.minor.patch` and make sure that still compares correctly, eg 2.3 < 2.3.1 < 2.3.2

Reply:
Great point - PR has been updated. Thanks @segiddins 👍